label_sync/labels.yaml

```
@@ -1019,6 +1019,19 @@ repos:
         target: prs
         prowPlugin: require-matching-label
         addedBy: prow
+      - color: e11d21
+        description: Indicates a PR lacks a `kind/foo` label and requires one.
+        name: do-not-merge/needs-kind
+        target: prs
+        prowPlugin: require-matching-label
+        addedBy: prow
+      - color: e11d21
+        description: Indicates an issue or PR lacks a `sig/foo` label and requires one.
+        name: do-not-merge/needs-sig
+        target: both
```

Oops looks like I didn't really click on the `Start Review` option in the UI :woman_facepalming: 

nit: do-not-merge doesn't have any meaning for issues :) 

```suggestion
        target: prs
```